dali/ft/daftformat.cpp

```diff
@@ -928,6 +928,21 @@ size32_t CCsvPartitioner::getSplitRecordSize(const byte * start, unsigned maxToR
         cur += matchLen;
     }
 
+    numOfProcessedBytes += (unsigned)(end - start);
+
+    if (isFirstRow)
+    {
+        // It seems there is only one row/record and it is terminated
+        // by EOF instead of TERMINATOR therefore the record structure definition
+        // generation doesn't finish yet.
+        isFirstRow = false;
+        LOG(MCdebugProgressDetail, unknownJob, "%d (0x%08x) bytes processed, but there is no terminator found. It seems there is only one row/record and it is terminated by EOF instead of TERMINATOR.", numOfProcessedBytes, numOfProcessedBytes);
```

Comment:
Based on the RFC 4180 1.2.2 it is valid CSV file. I put thsi logging only to record it is happened. Perhaps it is not necessary, so removed.